salt/modules/archive.py

```diff
@@ -466,6 +492,7 @@ def cmd_unzip(zip_file, dest, excludes=None,
     return _trim_files(files, trim_output)
 
 
+@salt.utils.decorators.depends('zipfile', fallback_function=cmd_unzip)
```

Comment:
Yeah, if you could open a PR against 2016.3 to remove it, that'd be great. Thanks.